### DISCUSSION TITLE
Allow Potential Adopters to Easily Spin up a registry using a Web Interface

### DISCUSSION THREAD

**Discussion by parthlawate** [Sunbird-RC/community#195] [Ideas & Enhancements]
For potential adopters wanting to take SBRC for a spin, it might be a good idea to create a small Web tool that allows them to supply the schema and spin up a registry quickly to try it out. 

Would something like this be of interest ? It will reduce the entry barrier & allow trying it out without having to actually do the technical setup.

**Comment by shankarmaruwada** [comment]
Excellent idea, Parth!

**Comment by pramodkvarma** [comment]
Also, we should have some community member to host demo of credentialing issuance as well. Pick a template or upload an SVG, generate 1 or many (CSV upload) certs based on that. Sort of a hosted issuer app demo. @parthlawate @gamemaker1 anyone up for it? I will start a new thread with @surendrasinghs on this.

**Reply by parthlawate** [reply]
Replied on the other thread. Work has started on it .. we'll accelerate on that front now.

**Comment by surendrasinghs** [comment]
@parthlawate 
Even better if we could seed it with a few sample registries... Imagine, 

"Would you like to spin up a registry?"

"Great, would you like to see a sample vehicle registry, blood donor registry, and so on OR are you ready to configure one for yourself?"

"Let's answer a few questions to get started with your own registry." 
[Questions: who is the registry for?, what about them would it contain?, ...]

**Reply by pramodkvarma** [reply]
Let us make both registry and VC issuance demo sandboxes up. Exciting!

**Reply by parthlawate** [reply]
@surendrasinghs Thats a good idea ! Will plan that. We've prioritised the VC Issuance one now. We'll take this up immediately after. @jayadinni @Renuka-S Pls note !

**Comment by SaketSinha** [comment]
Hi @parthlawate @surendrasinghs 

Can you point me to:
1. PRD
2. Project - Github
Interested in understanding the use case and requirements also the project progress that has been done till now.

Internally we at Samagra are also planning to work on a similar project.

cc - @ChakshuGautam

**Reply by parthlawate** [reply]
Hi @SaketSinha You need a PRD for which project ? Sunbird RC in general or for a Specific Use case ? 

Most of the documentation on each is open & available at https://docs.sunbirdrc.dev/ 

The Project Github is maintained under this same repo which can be seen here https://github.com/Sunbird-RC/community/projects?type=beta

**Reply by SaketSinha** [reply]
Hi @parthlawate - Thanks for responding, I am aware of the registry. 
As mentioned by you in the thread - "Allowing potential adopters to spin up a registry using a web interface".
We are also considering taking up a similar project to create. In this context, I want to catch up on the current thinking and progress of the web interface.

PRD - for the mentioned web interface if it exists, if not we can create one.
Project - Github project for the web interface. (We can contribute to the same)

**Comment by parthlawate** [comment]
Ah Ok :) 

Specific to this, while we have some high level ideas but a proper PRD is not yet in place.  Here's the current draft WIP version :  https://docs.google.com/document/d/1xMaTlgjlqZ_Q_fWk11SLH6AabQQT_b0vB5CkwMPOOn4/edit#heading=h.28pqdym5rzp3

Lets collaborate to detail this out.. 

A Project hasn't been created yet.. But we can go ahead and create it once

**Reply by SaketSinha** [reply]
Thanks for the info, I will check and get back.

**Reply by MaheshKulkarniTekdi** [reply]
@SaketSinha Hope you were able to discuss on the use case further.

**Reply by SaketSinha** [reply]
HI @MaheshKulkarniTekdi - Yes, I have had an internal discussion on this. I will be closing the PRD in a few days and sharing with the group for comments latest by early next week.
